compute_tools/src/pg_helpers.rs

```diff
@@ -123,7 +123,8 @@ impl RoleExt for Role {
         // XXX: consider putting LOGIN as a default option somewhere higher, e.g. in control-plane.
         // For now, we do not use generic `options` for roles. Once used, add
         // `self.options.as_pg_options()` somewhere here.
-        let mut params: String = "LOGIN".to_string();
+        let mut params: String = self.options.as_pg_options();
+        params.push_str(" LOGIN");
```

Comment:
Comment is out-of-date now.

Do we use the role's `options` field for something now?

Reply:
Yes, we use it to enable  CREATEDB and CREATEROLE when creating roles

Reply:
Removed the comment